src/app/onion/learning-object-builder/components/scaffold/add-child/add-child.component.ts

```diff
@@ -62,20 +62,8 @@ export class AddChildComponent implements OnInit, OnDestroy {
         });
       });
 
-    const releasedObjects = await this.searchLearningObjectService
-      .getLearningObjects({ ...filters, text: query, childId: this.child.id })
-      .then((response: { learningObjects: LearningObject[], total: number }) => {
-        let { learningObjects } = response;
-        const indx = this.lengths.indexOf(this.child.length);
-        const childrenLengths = this.lengths.slice(0, indx);
-        learningObjects = learningObjects.filter(
-          child => (!this.currentChildren.includes(child.id) && childrenLengths.includes(child.length))
-        );
-        return learningObjects;
-      });
-
```

Comment:
I would see if you could adjust the query to just include the author username. An author can add released learning objects as children of unreleased as long as they are the authors and are of a shorter length.